roles/baseline/tasks/main.yaml

```diff
@@ -45,6 +45,21 @@
   tags:
     - baseline
 
+- name: Lookup K8s version info
+  block:
+    - name: Retreive K8s cluster information
+      community.kubernetes.k8s_cluster_info:
+        kubeconfig: "{{ KUBECONFIG }}"
+      register: cluster_info
+    - name: Set the K8s server version
+      set_fact:
+        K8S_VERSION: "{{ cluster_info.version.server.kubernetes.major + '.' + cluster_info.version.server.kubernetes.minor }}"
+    - name: Output K8S_VERSION
+      debug:
+        msg: "{{ K8S_VERSION }}"
+  tags:
+    - baseline
```

Comment:
With this line the K8S_VERSION is only used in the `baseline` task during any action `install/uninstall` and not with any other task is that correct?

Reply:
That's the idea yes, right now that var is only referenced to make a decision for including the ebs-csi-driver task during baseline activity.